src/lib.rs

```diff
@@ -1,5 +1,5 @@
 extern crate chrono;
-extern crate crypto;
+extern crate ring;
```

Comment:
Pedantic: can you move this down to keep the list of crates in alphabetical order?